integration/localnet/bootstrap.go

```diff
@@ -42,6 +42,9 @@ const (
 	DefaultObserverCount     = 0
 	DefaultNClusters         = 1
 	DefaultProfiler          = false
+	DefaultTracing           = true
```

Comment:
I wonder if makes sense to have default tracing as false.

Reply:
This is a tricky one: i'm of course is quite biassed given that my workflow always includes traces.  If people do not use them on localnet, then I'm happy to switch the default.

FWIW, currently, it is hardcoded to `true` for localnet.